src/components/ContactsList.vue

```diff
@@ -157,6 +203,112 @@ export default {
 			}
 			return true
 		},
+		selectionChanged(newValue) {
+			if (this.selected.includes(newValue)) {
+				this.selected.splice(this.selected.indexOf(newValue), 1)
+			} else {
+				this.selected.push(newValue)
+			}
+		},
+		deleteMultipleContact() {
+			const temp = []
+			this.selected.forEach(element => {
+				if (this.contacts[element]) {
+					// delete contact
+					this.$store.dispatch('deleteContact', { contact: this.contacts[element] })
+					temp.push(this.selected.indexOf(element), 1)
+				}
+			})
+			// delete the uid in selected of the contact deleted
+			temp.forEach(el => {
+				this.selected.splice(temp, 1)
+			})
+		},
+		cleanContactValue(contact, value) {
+			contact.jCal[1].forEach(element => {
+				if (element[0] === value[0] && element[3] === '' && !Array.isArray(element[3])) {
+					contact.jCal[1].splice(contact.jCal[1].indexOf(element), 1)
+				} else if (element[0] === value[0] && Array.isArray(element[3])) {
+					let isempty = true
+					value[3].forEach(arr => {
+						if (arr !== '') {
+							isempty = false
+						}
+					})
+					if (isempty === false) {
+						contact.jCal[1].splice(contact.jCal[1].indexOf(element), 1)
+					}
+				}
+			})
+		},
+		addValue(contact, jcalvalue) {
+			jcalvalue.filter(element => {
+				// exclude the unique field we don't want to add
+				return !['uid', 'version', 'fn', 'prodid', 'gender', 'rev'].includes(element[0])
+			}).forEach(value => {
+				if (Array.isArray(value[3])) {
+					let isempty = true
+					value[3].forEach(arr => {
+						if (arr !== '') {
+							isempty = false
+						}
+					})
+					if (isempty === false) {
+						// delete blank field of the same type and push the new field
+						this.cleanContactValue(contact, value)
+						contact.jCal[1].push(value)
+					}
+				} else if (value[3] !== '') {
+					let include = false
+					contact.jCal[1].forEach(element => {
+						if (element[0] === value[0] && element[3] === value[3]) {
+							include = true
+						}
+					})
+					if (!include) {
+						// delete blank field of the same type and push the new field
+						this.cleanContactValue(contact, value)
+						contact.jCal[1].push(value)
+					}
+				}
+			})
+			return contact
+		},
+		mergeContact() {
+			const firstContact = this.contacts[this.selected[0]]
+			this.selected.slice(1).forEach((element) => {
+				if (this.contacts[element]) {
+					const contactjcal = this.contacts[element].jCal[1]
+					this.addValue(firstContact, contactjcal)
+					// delete the contact merged and the uid in the selected
+					this.$store.dispatch('deleteContact', { contact: this.contacts[element] }) && this.selected.splice(this.selected.indexOf(element), 1)
+				}
+			})
+			this.$store.dispatch('updateContact', firstContact)
+		},
+		OpenMultiselect() {
```

Comment:
```suggestion
		openMultiselect() {
```

camel case for methods